ios/RNGoogleAds/RNGoogleAdsCommon.h

```diff
@@ -46,6 +46,7 @@
 - (void)setRequestId:(NSNumber *)requestId;
 @end
 
+extern NSString *const GOOGLE_ADS_EVENT_APP_OPEN;
```

Comment:
```suggestion
extern NSString *const GOOGLE_MOBILE_ADS_EVENT_APP_OPEN;
```

since the lines are next to ones that changed this is going to be conflict city, dang